can/interfaces/socketcand/socketcand.py

```diff
@@ -75,10 +76,13 @@ def connect_to_server(s, host, port):
 
 
 class SocketCanDaemonBus(can.BusABC):
-    def __init__(self, channel, host, port, can_filters=None, **kwargs):
+    def __init__(self, channel, host, port, tcp_tune=False, can_filters=None, **kwargs):
```

Comment:
It would be nice if you could add a proper docstring like 
https://github.com/hardbyte/python-can/blob/38c4dc4b9ff2a932cdd1b776fb6a62df8a3e63b6/can/interfaces/socketcan/socketcan.py#L621

The socketcand interface API is not really well documented in `doc/interfaces/socketcand.rst`

Reply:
Alrighty, added docstrings to public methods of the Bus object.

Reply:
Awesome, thanks! You can see the docs for your PR [here](https://python-can--1683.org.readthedocs.build/en/1683/interfaces/socketcand.html), your changes do not show up yet.

You need to update [socketcand.rst](https://github.com/hardbyte/python-can/blob/develop/doc/interfaces/socketcand.rst).

Add something like
```
.. autoclass:: can.interfaces.socketcand.SocketCanDaemonBus
   :show-inheritance:
   :member-order: bysource
   :members:
```

You can build the docs locally with `python -m sphinx -Wan --keep-going doc build`.